.github/workflows/bump-version-flake.yml

```
@@ -0,0 +1,51 @@
+name: Bump nixpacks-flake version
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+env:
+  REGEX: v[0-9]+\.[0-9]+\.[0-9]+
+  FLAKE: flake.nix
+  BRANCH_NAME: auto-update/nixpacks-flake
+  GIT_AUTHOR_NAME: ${{ github.repository_owner }}
+  GIT_AUTHOR_EMAIL: ${{ github.repository_owner }}@users.noreply.github.com
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - name: Get the release version from the tag and old version
+        shell: bash
+        if: env.NIXPACKS_VERSION == ''
+        run: |
+          echo "NIXPACKS_VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          echo "version is: ${{ env.NIXPACKS_VERSION }}"
```

Should just be this since the value isn't on `env` yet

```suggestion
          echo "version is: ${{ NIXPACKS_VERSION }}"
```